changelog.d/20251204_140428_chris_configure_endpoint_config_sc_46072.rst

```diff
@@ -0,0 +1,12 @@
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- Added two options to ``globus-compute-endpoint configure``, ``--manager-config``
+  and ``--template-config``, which allow specifying alternate configuration files
+  to be copied into the new endpoint directory during configuration.
+
+Deprecated
+^^^^^^^^^^
+
+- Deprecated the ``--endpoint-config`` option of ``globus-compute-endpoint configure``,
+  in favor of the new ``--manager-config`` and ``--template-config`` options.
```

Comment:
It's not deprecated so much as it's an outright error to use it now&nbsp;&mdash;&nbsp;`raise ClickException(...)`!

Which is actually probably fine at this point: we did the bull-headed thing of breaking it in v4 already, so, the damage is done.  At this point, could probably just remove it entirely.